src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -115,5 +115,101 @@
+      AdditionalDeploymentContributorPaths;
+      AdditionalDeploymentContributors;
+      AllowDropBlockingAssemblies;
+      AllowIncompatiblePlatform;
+      AllowUnsafeRowLevelSecurityDataMovement;
+      BackupDatabaseBeforeChanges;
+      BlockOnPossibleDataLoss;
+      BlockWhenDriftDetected;
+      CommandTimeout;
+      CommentOutSetVarDeclarations;
+      CompareUsingTargetCollation;
+      CreateNewDatabase;
+      DatabaseLockTimeout;
+      DatabaseSpecification;
+      DeployDatabaseInSingleUserMode;
+      DisableAndReenableDdlTriggers;
+      DoNotAlterChangeDataCaptureObjects;
+      DoNotAlterReplicatedObjects;
+      DoNotDropObjectTypes;
+      DropConstraintsNotInSource;
+      DropDmlTriggersNotInSource;
+      DropExtendedPropertiesNotInSource;
+      DropIndexesNotInSource;
+      DropObjectsNotInSource;
+      DropPermissionsNotInSource;
+      DropRoleMembersNotInSource;
+      DropStatisticsNotInSource;
+      ExcludeObjectTypes;
+      GenerateSmartDefaults;
+      IgnoreAnsiNulls;
+      IgnoreAuthorizer;
+      IgnoreColumnCollation;
+      IgnoreColumnOrder;
+      IgnoreComments;
+      IgnoreCryptographicProviderFilePath;
+      IgnoreDdlTriggerOrder;
+      IgnoreDdlTriggerState;
+      IgnoreDefaultSchema;
+      IgnoreDmlTriggerOrder;
+      IgnoreDmlTriggerState;
+      IgnoreExtendedProperties;
+      IgnoreFileAndLogFilePath;
+      IgnoreFilegroupPlacement;
+      IgnoreFileSize;
+      IgnoreFillFactor;
+      IgnoreFullTextCatalogFilePath;
+      IgnoreIdentitySeed;
+      IgnoreIncrement;
+      IgnoreIndexOptions;
+      IgnoreIndexPadding;
+      IgnoreKeywordCasing;
+      IgnoreLockHintsOnIndexes;
+      IgnoreLoginSids;
+      IgnoreNotForReplication;
+      IgnoreObjectPlacementOnPartitionScheme;
+      IgnorePartitionSchemes;
+      IgnorePermissions;
+      IgnoreQuotedIdentifiers;
+      IgnoreRoleMembership;
+      IgnoreRouteLifetime;
+      IgnoreSemicolonBetweenStatements;
+      IgnoreTableOptions;
+      IgnoreTablePartitionOptions;
+      IgnoreUserSettingsObjects;
+      IgnoreWhitespace;
+      IgnoreWithNocheckOnCheckConstraints;
+      IgnoreWithNocheckOnForeignKeys;
+      IncludeCompositeObjects;
+      IncludeTransactionalScripts;
+      LongRunningCommandTimeout;
+      NoAlterStatementsToChangeClrTypes;
+      PopulateFilesOnFileGroups;
+      RegisterDataTierApplication;
+      RunDeploymentPlanExecutors;
+      ScriptDatabaseCollation;
+      ScriptDatabaseCompatibility;
+      ScriptDatabaseOptions;
+      ScriptDeployStateChecks;
+      ScriptFileSize;
+      ScriptNewConstraintValidation;
+      ScriptRefreshModule;
+      SqlCommandVariableValues;
+      TreatVerificationErrorsAsWarnings;
+      UnmodifiableObjectWarnings;
+      VerifyCollationCompatibility;
+      VerifyDeployment;
+    </KnownDeployProperties>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Define some useful defaults for deployment -->
+    <TargetServerName>localhost</TargetServerName>
```

Comment:
Should this not be: (local)

Reply:
Hmm, not sure. I don't know if that works cross-platform while I'm fairly sure that localhost works everywhere. Then again, I think `(local)` is more idiomatic SQL so I'll test to see if it works cross-platform.

Reply:
Looks like that works on Mac as well. I only needed to add some quotes around the server name, but I guess that's a good thing to do anyway.